README.md

```diff
@@ -1,11 +1,16 @@
 # Swagger Assert4J - a Java library for API testing
 
-The Assert4J library lets you test APIs through Java, Groovy or Cucumber. The library has extensive support for REST, SOAP, JDBC and JMS protocols. Under the hood the library uses the test-execution engine of [SoapUI](http://www.soapui.org).
-
-Read on to get started
-* [with Java](#getting-started-with-java) - together with any testing framework
-* [with Groovy](#getting-started-with-groovy) - together with any testing framework 
-* [with Cucumber](modules/cucumber) - with cucumber for java 
+The Assert4J library lets you test APIs through Java, Groovy or Cucumber. The library has extensive support for REST, SOAP, JDBC and JMS protocols. 

```

Comment:
Unfortunately I don't think the library has support JMS test steps yet. It probably should have though.